pandas/_testing.py

```diff
@@ -1181,6 +1193,7 @@ def assert_series_equal(
             check_less_precise=check_less_precise,
             check_dtype=check_dtype,
             obj=str(obj),
+            index=np.asarray(left.index),
```

Comment:
the docstring in assert_almost equal says this is a string or None

Reply:
Yes it does :-(
Sorry - now fixed - thanks.